dbms/src/Storages/Page/PageUtil.h

```diff
@@ -77,6 +77,7 @@ extern const int FILE_SIZE_NOT_MATCH;
 
 namespace FailPoints
 {
+extern const char skip_check_segment_update[];
```

Comment:
This failpoint is enabled in https://github.com/pingcap/tiflash/blob/9407b18b3dd909a5ef3895ac01abb8d9ea163b83/dbms/src/Storages/DeltaMerge/tests/MultiSegmentTestUtil.h#L66 so that the splitted segment keeps stable and will not get merged or splitted again.

Reply:
Oh, good catch!